src/Cake.Svn/SvnAliases.Add.cs

```diff
@@ -1,51 +1,113 @@
 ﻿using Cake.Core;
 using Cake.Core.Annotations;
+using Cake.Core.IO;
 using Cake.Svn.Add;
 
 namespace Cake.Svn
 {
     public static partial class SvnAliases
     {
         /// <summary>
-        /// Add a file or directory to Subversion.
+        /// Add a directory to Subversion.
         /// </summary>
         /// <param name="context">The Cake context.</param>
-        /// <param name="fileOrDirectoryToAdd">The filename or directoryname to add.</param>
-        /// <returns>True if the command were successful. Otherwise False will be returned.</returns>
+        /// <param name="directory">The directory.</param>
+        /// <returns>
+        /// <c>true</c> if the command were successful. Otherwise <c>false</c> will be returned.
```

Comment:
`Returns <c>true</c> if the command was successful. Otherwise <c>false</c> will be returned.`